libs/damap/src/lib/widgets/flip-card/flip-card.component.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+
+import { FlipCardComponent } from './flip-card.component';
+import { MatCardModule } from '@angular/material/card';
+import { MatIconModule } from '@angular/material/icon';
+import { Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
```

Comment:
`RouterTestingModule` is deprecated

```suggestion
import { Router, RouterModule } from '@angular/router';
```